static/sass/custom/_search-panel.scss

```diff
@@ -1,16 +1,16 @@
 .p-search-panel {
   background: $color-x-light;
-  z-index: 999;
   box-shadow: 0 1px 5px 1px transparentize($color-dark, 0.8);
   left: 0;
   margin-top: 0;
   opacity: 0;
   padding: $sp-medium;
   position: absolute;
+  pointer-events: none;
   top: 3rem;
   transition: opacity 0.25s;
   width: 100%;
-  z-index: -1;
+  z-index: -9999;
```

Comment:
We should do some z-index sorting here like we do in the GUI....anyways that can be done in a follow-up